client/src/components/Home/Home.test.tsx

```diff
@@ -29,7 +29,7 @@ describe('<Home />', () => {
 
     it('has a reading list', () => {
       renderHome(initialAuthState);
-      expect(screen.getByText(/Reading List/)).toBeInTheDocument();
+      expect(screen.getAllByText(/Reading List/)).toBeDefined();
```

Comment:
This is good as-is, but if you want more fine-grained assertions, you can update it to
```ts
expect(screen.getAllByText(/Reading List/)).toHaveLength(2);
```